Makefile

```diff
@@ -20,50 +20,29 @@ gen-lang: require-jinja2
 	bin/gen-lang.py
 
 build: gen-lang
-	rm -rf .build build && bin/build.py # && cp -R fanqiang build/
+	rm -rf .build build && bin/build.py
 
 run: build
 	cd build && python -m SimpleHTTPServer
 
-copy-installers: require-secrets-dir
```

Comment:
The feature is handled in `lantern-build` when releasing new versions. Code here is no longer used.